packages/astro/src/core/config.ts

```diff
@@ -1,15 +1,50 @@
 import type { AstroConfig, AstroUserConfig, CLIFlags } from '../@types/astro';
 import type { Arguments as Flags } from 'yargs-parser';
+import type * as Postcss from 'postcss';
 
 import * as colors from 'kleur/colors';
 import path from 'path';
 import { pathToFileURL, fileURLToPath } from 'url';
 import { z } from 'zod';
 import load from '@proload/core';
 import loadTypeScript from '@proload/plugin-tsm';
+import postcssrc from 'postcss-load-config';
```

Comment:
Not a complaint with you but it severely bums me out that `postcss-load-config` only supports `cjs` files.

Reply:
😫